File: LP_API/LP_API/Model/Game.cs
namespace LP_API.Model
{
    public class Game
    {
        public string Id { get; set; }
        public string GroupId { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public string Location { get; set; }
        public int Capacity { get; set; }
        public string Sport { get; set; }
        public bool Active { get; set; }
        public int EventId { get; set; }
    }

    public class GameResponse
    {
        public string Status { get; set; }
        public List<Game> Games { get; set; }
    }
}

File: Projeto/Lp3_Grupo5/src/main/java/com/lp3_grupo5/lp3_grupo5/Model/Game.java
package com.lp3_grupo5.lp3_grupo5.Model;

public class Game {
    private String Id;
    private String GroupId;
    private String StartDate;
    private String EndDate;
    private String Location;
    private int Capacity;
    private String Sport;
    private boolean Active;
    private int EventId;

    // Getters and setters
    public String getId() {
        return Id;
    }

    public void setId(String id) {
        Id = id;
    }

    public String getGroupId() {
        return GroupId;
    }

    public void setGroupId(String groupId) {
        GroupId = groupId;
    }

    public String getStartDate() {
        return StartDate;
    }

    public void setStartDate(String startDate) {
        StartDate = startDate;
    }

    public String getEndDate() {
        return EndDate;
    }

    public void setEndDate(String endDate) {
        EndDate = endDate;
    }

    public String getLocation() {
        return Location;
    }

    public void setLocation(String location) {
        Location = location;
    }

    public int getCapacity() {
        return Capacity;
    }

    public void setCapacity(int capacity) {
        Capacity = capacity;
    }

    public String getSport() {
        return Sport;
    }

    public void setSport(String sport) {
        ;
    }

    public boolean isActive() {
        return Active;
    }

    public void setActive(boolean active) {
        Active = active;
    }

    public int getEventId() {
        return EventId;
    }

    public void setEventId(int eventId) {
        EventId = eventId;
    }
}